src/router/index.ts

```diff
@@ -198,6 +198,10 @@ export const portalRouter = () => {
 
   // check is authenticated developer
   router.beforeEach(async (to, from, next) => {
+    if (to.meta.public) {
```

Comment:
question: will all routes have a `meta` even if it is not present in the definition, e.g. for custom created routes?

Reply:
answer: indeed